tests/test_pagination.py

```diff
@@ -57,6 +57,13 @@ def pagination_test(self, conn_id, testable_streams):
         actual_fields_by_stream = runner.examine_target_output_for_fields()
 
         for stream in testable_streams:
+
+            # skip 'locations' stream as there is not much info about
+            # limit of records returned in 1 page
+            # Documentation: https://help.shopify.com/en/manual/locations/setting-up-your-locations
+            if stream in ['locations']:
```

Comment:
Can we remove the location stream from the `testable_streams`. Remove this from line no. 23

Reply:
Removed _locations_ from `testable_streams` as per suggestion.